packages/core/src/controllers/SendController.ts

```diff
@@ -225,15 +225,17 @@ export const SendController = {
         params.receiverAddress &&
         params.tokenAddress
       ) {
+        const tokenAddress = CoreHelperUtil.getPlainAddress(
+          params.tokenAddress as CaipAddress
+        ) as `0x${string}`
```

Comment:
We should move `0x${string}` type definition to `common/TypeUtil.ts` and remove from all the duplications in the code - but not for this PR